bin/branding-files.list

```diff
@@ -1 +1,2 @@
-DESCRIPTION_FILE
\ No newline at end of file
+DESCRIPTION_FILE
+DESCRIPTION_FILE_DEB
```

Comment:
👎  on adding another variable to the branding list -- this already has entirely too many variables to set.  

Better to do transformation on the description file content  in the Debian build.sh (literally fold & sed commands to create a modified *version* of the file).  We can redefine the ENV variable to point at the modified file before calling the branding.py script, and then re-set it after.

**IMPORTANT** at no point do we want to try to read the file content into make variables.  They mangle it horribly.  File --> file transformation via a bash command is safer.

(Explained more about this in person).